src/mongo/MongoScrapbook.ts

```diff
@@ -243,18 +243,24 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 		if (argumentsContext) {
 			let functionCallContext = argumentsContext.parent;
 			if (functionCallContext && functionCallContext.parent instanceof mongoParser.CommandContext) {
-				const lastCommand = this.commands[this.commands.length - 1];
-				const argAsObject = this.contextToObject(ctx);
-				const argText = EJSON.stringify(argAsObject);
-				lastCommand.arguments.push(argText);
-				let escapeHandled = this.deduplicateEscapesForRegex(argText);
-				let ejsonParsed = {};
 				try {
-					ejsonParsed = EJSON.parse(escapeHandled);
-				} catch (err) { //EJSON parse failed due to a wrong flag, etc.
+					//tslint:disable:no-var-self
+					const me = this;
```

Comment:
If you're going to store the current object context in another variable, it would make more sense for the name to be discernible - say visitor.

Reply:
"me = this" is a well-established JavaScript pattern